Sources/Public/Animation/LottieAnimationView.swift

```diff
@@ -178,6 +178,221 @@ final public class LottieAnimationView: LottieAnimationViewBase {
     commonInit()
   }
 
+  // MARK: Open
+
+  /// Sets the current animation time with a Progress Time
+  ///
+  /// Note: Setting this will stop the current animation, if any.
+  /// Note 2: If `animation` is nil, setting this will fallback to 0
+  open var currentProgress: AnimationProgressTime {
```

Comment:
I see the rationale of wanting to override the `play` methods to provide custom functionality. What use case do you have in mind for overriding `currentProgress` / `currentTime` / `currentFrame`? What sort of additional logic would you need to layer on top of these?

Reply:
My also allowing these to be overridden was speculative and can be left out of this PR. We aren't actually overriding these at the moment, I was just anticipating that we might want to in the future.

For example, if we are ignoring play requests (e.g. in response to reduce motion) and keeping the animation static, it might also makes sense to ignore changes to the current frame to avoid the animation jumping. We don't have a concrete case like this yet though.

Reply:
Cool, in that case I'd prefer if we left these properties as just `public`. Totally fine with marking the various `play` methods as open though!